### DISCUSSION TITLE
V免签支付失败

### DISCUSSION THREAD

**Discussion by 2694151619** [Baiyuetribe/kamiFaka#139] [Bug反馈]
我后台配置好了 在前端显示暂未发现支付接口   支付也是失败
![{J_KP{7SSC5D6}{RY{QR%`3](https://user-images.githubusercontent.com/96507662/162017432-38302708-9cab-4727-91b0-f71620a7cd2e.png)
![DPZKF5L J D2M1AH0~R`) O](https://user-images.githubusercontent.com/96507662/162017507-167c3476-4e10-4a16-b9ae-ef24598a8c5f.png)

**Comment by 2694151619** [comment]
能支付 能收款 但就是不跳转 显示未支付 但是已经支付

**Comment by Yusheng-X** [comment]
我在使用云免签支付接口时遇到了相同的问题
